src/components/structures/SpaceHierarchy.tsx

```diff
@@ -210,6 +210,25 @@ const Tile: React.FC<ITileProps> = ({
         topic = room.topic;
     }
 
+    let topicSection: ReactNode | undefined;
+    if (topic) {
+        topicSection = (
+            <Linkify
+                options={{
+                    attributes: {
+                        onClick(ev: MouseEvent) {
+                            // prevent clicks on links from bubbling up to the room tile
+                            ev.stopPropagation();
+                        },
+                    },
+                }}
+            >
+                {" · "}
```

Comment:
~~Previously we had a dot only if there was no topic but here we always add one. Is that a deliberate change?~~

Nope. Previously the code confused me and now it's clearer :-)